python/langsmith/client.py

```diff
@@ -491,6 +504,20 @@ def __init__(
         # Create a session and register a finalizer to close it
         session_ = session if session else requests.Session()
         self.session = session_
+        self.compress_traces = ls_utils.get_env_var("USE_RUN_COMPRESSION")
+        if self.compress_traces:
+            self._futures: set[cf.Future] = set()
+            self._boundary = BOUNDARY
```

Comment:
If this is a constant why do we need a whole slot property just for this?

Reply:
fixed